pkg/apis/batch/types.go

```diff
@@ -138,6 +138,27 @@ const (
 	PodFailurePolicyOnExitCodesOpNotIn PodFailurePolicyOnExitCodesOperator = "NotIn"
 )
 
+// This field controls when we recreate pods
```

Comment:
It isn't a field here but an enum-like type definition. Looking at other type definitions I think it is enough to say:
```
// PodReplacementPolicy specifies the policy for creating pod replacements.
// +enum
```

Reply:
resolve as API review is in another PR.